server/src/main/java/org/opensearch/action/search/SearchQueryCounters.java

```diff
@@ -21,20 +21,36 @@ final class SearchQueryCounters {
     // Counters related to Query types
     public final Counter aggCounter;
     public final Counter boolCounter;
+    public final Counter boostCounter;
+    public final Counter constantScoreCounter;
+    public final Counter disMaxCounter;
+    public final Counter distanceFeatureCounter;
+    public final Counter existsCounter;
+    public final Counter fieldMaskingSpanCounter;
     public final Counter functionScoreCounter;
+    public final Counter fuzzyCounter;
+    public final Counter geoBoundingBoxCounter;
+    public final Counter geoDistanceCounter;
+    public final Counter geoPolygonCounter;
+    public final Counter geoShapeCounter;
+    public final Counter intervalCounter;
     public final Counter matchCounter;
+    public final Counter matchallCounter;
     public final Counter matchPhrasePrefixCounter;
     public final Counter multiMatchCounter;
     public final Counter otherQueryCounter;
+    public final Counter prefixCounter;
```

Comment:
The naming for these query counters is inconsistent. We should stick to without Query suffix nomenclature. Few of the violations are `queryStringQueryCounter`(additional Query), `boostCounter`(missing ing), `regexCounter` (missing p)